### ISSUE TITLE
Halo Reach Engine version

### ISSUE THREAD

**Issue by AxioParadox** [XboxChaos/Assembly#334]
**Describe the bug**
Assembly states when attempting to open a campaign map that version may 26 2021 is not supported which would also imply that the Jun 23 version would also not be supported. I attempted to add both versions to the engines.xml and the program does not even start now. 

**PC (please complete the following information):**
 - OS: Windows 10

**Comment by A267-dev** [comment]
You haven't added the Proper Plugins for your Added Engine Version.

**Comment by ChicagoFaucet** [comment]
I'm having the same issue.  ELI5:  proper plugins from where to where?

![image](https://user-images.githubusercontent.com/95878859/145453301-96cd6c27-20d5-4204-a2dd-dd6cab5cf8a1.png)

**Comment by A267-dev** [comment]
> I'm having the same issue. ELI5: proper plugins from where to where?
> 
> ![image](https://user-images.githubusercontent.com/95878859/145453301-96cd6c27-20d5-4204-a2dd-dd6cab5cf8a1.png)

What Build of Reach MCC are you Trying to Load?

**Comment by ChicagoFaucet** [comment]
Please excuse my ignorance.  I am new to modding.  I'm actually trying to learn it right now, so if my questions or answers seem stupid, I'm learning.

I don't know the most direct answer to your question.  I can tell you that I am trying to use the most current downloaded version through Steam.

![image](https://user-images.githubusercontent.com/95878859/145587288-1719bb5f-b8f6-4aef-861d-aa483a578b7f.png)
![image](https://user-images.githubusercontent.com/95878859/145587697-d20aa6d8-548e-4c5d-b075-50d8628eb6c0.png)

**Comment by A267-dev** [comment]
> Please excuse my ignorance. I am new to modding. I'm actually trying to learn it right now, so if my questions or answers seem stupid, I'm learning.
> 
> I don't know the most direct answer to your question. I can tell you that I am trying to use the most current downloaded version through Steam.
> 
> ![image](https://user-images.githubusercontent.com/95878859/145587288-1719bb5f-b8f6-4aef-861d-aa483a578b7f.png) ![image](https://user-images.githubusercontent.com/95878859/145587697-d20aa6d8-548e-4c5d-b075-50d8628eb6c0.png)

I see the issue, you should be running the Dev branch of assembly instead of the main branch.

**Comment by ChicagoFaucet** [comment]
Naw, that didn’t work.  I tried it both ways.  Still the same error message.

Plus, I’m trying this guy’s route, and he says to not use the Dev load:

https://www.youtube.com/watch?v=-6gTRi2kkhU
